README.md

```diff
@@ -1,6 +1,58 @@
 ![Logo Dark](https://github.com/camunda/issues/blob/main/resources/Logo_White_Github.png#gh-dark-mode-only)
 ![Logo Light](https://github.com/camunda/issues/blob/main/resources/Logo_Black.png#gh-light-mode-only)
 
-# Welcome to Camunda Issues 
+# Welcome to Camunda Public Roadmap 
 
-In this repository you can get an overview of key features we are currently working on (or planning to work on in the near future). Please don't hesitate to comment any questions or feedback you have directly on the respective issues.
+❇️ View the official [Camunda public product roadmap](https://github.com/orgs/camunda/projects/57).
+
+Our product roadmap is where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you. Have any questions or comments about items on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
+
+The issues repository is for communicating Camunda’s roadmap. If you have feedback about the issues repository itself, such as how the issues are presented, let us know through general feedback using the appropriate GitHub issues template.
```

Comment:
What do you want to express here? Shall the users create new issues for "meta-feedback"?

Reply:
The intent here is to enable the user to provide feedback as much on the product as on the process to share the feedback itself. We want to continuously improve the way we interact with them and this feedback is important.